packages/checkout/widgets-lib/src/locales/en.json

```diff
@@ -234,6 +234,12 @@
         "title": "How would you like to add tokens?"
       },
       "topUpOptions": {
+        "onramp": {
+          "heading": "Buy tokens",
+          "caption": "With card, Google pay or Apple pay available.",
+          "subcaption": "Fees ",
+          "disabledCaption": ""
```

Comment:
```suggestion
          "heading": "Buy tokens"
```

Reply:
these other options are not required, because are not being used

Reply:
updated